html-web-page/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <!-- your HTML code here 👇 -->
+  <h1>Lorem Ipsum of the Month Club</h1>
+  <h2>Make all your friends lorem of your ipsum!</h2>
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam, perspiciatis delectus facilis sequi veritatis perferendis error minima cumque! Voluptates veniam ea fuga veritatis commodi quo illum, facere labore natus suscipit adipisci maiores blanditiis, minima vel laudantium dolores libero excepturi alias inventore consectetur.</p>
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Lorem Ipsum Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new lorums curated per month</td>
+        <td>&#10003;</td>
+        <td>&#10003;</td>
+        <td>&#10003;</td>
+      </tr>
+      <tr>
+        <td>Monthly lorems come with commemorative ipsum jars</td>
+        <td></td>
+        <td>&#10003;</td>
+        <td>&#10003;</td>
+      </tr>
+      <tr>
+        <td>Free bonus ipsum jar of preserves bi-monthly</td>
+        <td></td>
+        <td>&#10003;</td>
+        <td>&#10003;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to LoremCon</td>
+        <td></td>
+        <td></td>
+        <td>&#10003;</td>
+      </tr>
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <p>Like what you see? Become a part of the LIOTM family today!</p>
+  <div>
+    <form>
+      <label for="name">
+        <input type="text" id="name" placeholder="Full Name">
+      </label>
+      <label for="plan-select">
```

Comment:
There is no text labeling this form control element, so no `label` element is necessary